force-app/main/default/classes/SOQL.cls

```diff
@@ -201,7 +277,14 @@ public with sharing class SOQL {
      * @return A list of Cases
      */
     public static List<Case> getCasesWithStatus() {
-        return null; // Replace null with the variable you used to store the result
+        List <Case> casesWithStatus = [
+            SELECT Status, CreatedDate, Id
+            FROM Case
+            WHERE status = 'open'
```

Comment:
Try to filter by not closed since some statuses on Case might be considered open even if the status does not equal to the string 'Open'. An example of this is if a case's status is 'New'